lib/cocina/models/admin_policy.rb

```diff
@@ -7,6 +7,8 @@ class AdminPolicy < Struct
 
       include Checkable
 
+      include TitleBuilder
```

Comment:
I don't think we should include these behaviors on the models.  Can these just be services?

Reply:
Yeah - totally, I was actually thinking that would be a good idea but would like a short pairing to discuss.